packages/auth/src/accessToken/service.ts

```diff
@@ -85,3 +99,31 @@ async function revoke(
     return new Error('token not found')
   }
 }
+
+async function createAccessToken(
+  deps: ServiceDependencies,
+  grantId: string,
+  opts?: AccessTokenOpts
+): Promise<AccessToken> {
+  const invTrx = opts?.trx || (await AccessToken.startTransaction(deps.knex))
```

Comment:
Why is that called `invTrx`? 
I actually don't think we have to do the create + commit approach here because there is nothing else going on at the same time. We only create the access token and don't call on another service that could cause problems in which case we don't want to commit the transaction. 
So you could just use `AccessToken.query(deps.knex).insertAndFetch`.